wmn-data.json

```diff
@@ -2912,19 +2912,14 @@
        },
        {
         "name": "LeetCode",
-        "uri_check": "https://leetcode.com/graphql",
+        "uri_check": "https://alfa-leetcode-api.onrender.com/{account}",

```

Comment:
We are trying to move away from relying on 3rd party sites to check other sites for us. I'm pretty sure onrender.com is not affiliated with leetcode.com except it allows you to run code. Right?

Reply:
Yeah, it's a third party API. The leetcode.com GraphQL endpoint isn't properly documented and is known to throw 403 issues due to cloudflare, 

source: https://www.reddit.com/r/leetcode/comments/18sflv5/leetcode_graphql_endpoint/

Reply:
This is unfortunate for us. Unless you can find a more native endpoint/URL to check, I do not think this can be a part of WMN.

Reply:
I'll close the PR for now then, I tried with more native URLs and I'm getting a 403.